backend/cesium_api/__main__.py

```diff
@@ -82,6 +82,7 @@ def main():
         port=args.port,
         api_prefix=api_prefix,
         data_dir=args.data_dir,
+        tiles_dir=DEFAULT_TILES_DIR,
```

Comment:
```suggestion
```

See related comment below about `DEFAULT_DATA_DIR` vs `DEFAULT_TILES_DIR`.

But make sure to remove this here.